amaranth/vendor/xilinx.py

```diff
@@ -341,17 +342,45 @@ def vendor_toolchain(self):
 
     # Symbiflow templates
 
-    _symbiflow_part_map = {
-        "xc7a35ticsg324-1L": "xc7a35tcsg324-1", # Arty-A7
-    }
+    # symbiflow does not distinguish between speed grades
+    # TODO: join with _xray_part
+    @property
+    def symbiflow_part(self):
```

Comment:
I think we don't necessarily need to export these as public names, unless we commit to maintaining them as a stable API. Could you make them private, please?

Reply:
Done.